### DISCUSSION TITLE
Can't add credentials on SMTP endpoint

### DISCUSSION THREAD

**Discussion by arthurzenika** [postalserver/postal#2639] [Feature suggestions]
## Describe the bug

When adding an SMTP Endpoint using the WebUI I can't find any options to add credentials for an authenticated SMTP destination

![image](https://github.com/postalserver/postal/assets/445200/beedd1e9-6e19-4bcf-b803-2ac1bac4360f)

## To Reproduce

1. Go to /org/example/servers/example/smtp_endpoints
2. Click on 'Add another SMTP Server'
3. No fields in the form to add authentication
4. Even after creation no possibility to add credentials.

## Expected behaviour

Have some optionnal configuration for Authentication in SMTP endpoints

## Environment details 

 - OS: n/a
 - Browser : n/a
 - Version : 2.1.4
 - Type : n/a

**Comment by willpower232** [comment]
unfortunately that is not currently a feature, are you trying to relay messages out of Postal or do you have an SMTP receiving service you'd like Postal to pass messages it receives to.

**Comment by MSabino85** [comment]
This functionality would be very useful and secure.
To the smtp relay, it allows some protection against **spoofing**.
It also allows application and database servers to send mail from an internal relay to an external one, applying access rules by IP/credentials, allowing certain sending addresses, mapping to end points (http or smtp) for each destination/sending address.
In my case the mailtrap service requests authentication.

**Comment by cgrauls** [comment]
i also would like to have this functionality. Is there a way we can manual add this authentication arround the UI?
